File: lecture_7_string/checkApperance.cpp
#include<bits/stdc++.h>
using namespace std;

void checkApperance(char n[1001]){
    bool flag = true;
    for(int i =0; i < strlen(n);i++){
        n[i]=toupper(n[i]);
        if(n[i] == 'B' || n[i] == 'I'||n[i] == 'O'||n[i] == 'G'){
            flag = true;
            break;
        }else{
            flag = false;
        }
    }
    if(flag == true){
        cout <<"YES";
    }else{
        cout <<"NO";
    }
}

int main(){
char n[1001];
cin.getline(n,1001);
checkApperance(n);
    return 0;
}

File: lecture_7_string/checkApperance.py
n = input()
flag = True
for i in range(len(n)):
    n = n.upper()
    if (n[i] == 'B' or n[i] == 'I' or n[i] == 'G' or n[i] == 'O'):
        flag = True
        break
    else:
        flag = False

if(flag == True):
   print("YES")
else:
    print("NO")
